src/StructuralModel/StructuralMaterials.jl

```diff
@@ -41,7 +46,7 @@ end
 StructuralMaterial(vmats::AbstractMaterial...) = StructuralMaterial(collect(vmats))
 
 "Return the `Material` mapped with the label `l`."
-function Base.getindex(sm::StructuralMaterial, l::L) where {L<:Union{Symbol,AbstractString}}
+function Base.getindex(sm::StructuralMaterial, l::Label)
```

Comment:
this change is unrelated -- goes in the direction of normalizing `Label` into just `Symbol` at a later stage, so we want to apply `Symbol` earlier in the pipeline.

Reply:
We can try, but MshReader or Gmsh might have an issue with 'Symbol' labels 🏷